Messaging.md

```diff
@@ -0,0 +1,63 @@
+# Messaging Protocol
+
+## Overview
+
+This protocol assumes an underlying authenticated and ordered transport mechanism that takes care of framing individual messages.
+[BOLT #8](08-transport.md) specifies the canonical transport layer used in Lightning, though it can be replaced by any transport that fulfills the above guarantees.
+
+All data fields are unsigned big-endian unless otherwise specified.
+
+## Table of Contents
+
+* [Connection Handling and Multiplexing](#connection-handling-and-multiplexing)
+  * [Message Format](#message-format)
+  * [Fundamental Types](#fundamental-types)
+  * [Authors](#authors)
+
+## Connection Handling and Multiplexing
+
+Implementations MUST use a single connection per peer; contract messages (which include a contract ID) are multiplexed over this single connection.
+
+## Message Format
+
+We reuse the [Lightning Message Format](01-messaging.md#lightning-message-format) and the [Type-Length-Value Format](01-messaging.md#type-length-value-format)
+
+## Fundamental Types
+
+Various fundamental types are referred to in the message specifications:
+
+* `byte`: an 8-bit byte
+* `u16`: a 2 byte unsigned integer
+* `u32`: a 4 byte unsigned integer
+* `u64`: an 8 byte unsigned integer
+
+Inside TLV records which contain a single value, leading zeros in
+integers can be omitted:
+
+* `tu16`: a 0 to 2 byte unsigned integer
+* `tu32`: a 0 to 4 byte unsigned integer
+* `tu64`: a 0 to 8 byte unsigned integer
+
+The following convenience types are also defined:
+
+* `chain_hash`: a 32-byte chain identifier (see [BOLT #0](00-introduction.md#glossary-and-terminology-guide))
+* `contract_id`: a 32-byte contract_id (see [Protocol Specification](Protocol.md))
+* `sha256`: a 32-byte SHA2-256 hash
+* `signature`: a 64-byte bitcoin Elliptic Curve signature
+* `point`: a 33-byte Elliptic Curve point (compressed encoding as per [SEC 1 standard](http://www.secg.org/sec1-v2.pdf#subsubsection.2.3.3))
+* `spk`: A bitcoin script public key encoded as ASM prefixed with a CompactSize
+* `short_contract_id`: an 8 byte value identifying a contract (see [BOLT #7](07-routing-gossip.md#definition-of-short-channel-id))
+* `bigsize`: a variable-length, unsigned integer similar to Bitcoin's CompactSize encoding, but big-endian.  Described in [BigSize](01-messaging.md#appendix-a-bigsize-test-vectors).
+* `contract_info`: ???
```

Comment:
Yes, that is my intention (https://github.com/discreetlogcontracts/dlcspecs/pull/59#discussion_r470900331) not really sure where that stuff should go so I just put that here for now, when a PR opens that specifies v0 of this this will likely move :)